### ISSUE TITLE
Need advice: Still high temps. What RPM is good?

### ISSUE THREAD

**Issue by plimmplamm** [patrickcollins12/esphome-fan-controller#29]
Thanks for this great project first of all. 

Can I ask you all what RPM you are using?
My 6RU Server rack is at 40 degree celsius with 100% voltage and 1150 RPM.
Might have to use a stronger fan.

Also, what causes the fan to run at 1150 RPM even though it can do 1500?

Thanks for your hep

**Comment by patrickcollins12** [comment]
yes, add a 2nd or 3rd fan to move more air and check the ventilation in and out. 

As for the lower RPM it's possibly because you haven't removed my 80% max limit that I added to prevent my fans getting too noisy. 
![image](https://github.com/patrickcollins12/esphome-fan-controller/assets/942296/e73fb24b-6553-4f68-983f-c8eff1935264)
